frontend/src/components/layout/NavBar.jsx

```diff
@@ -63,7 +63,7 @@ class NavBar extends React.Component {
   constructor(props) {
     super(props)
 
-    this.DEBUGGING = false
+    this.DEBUGGING = true
```

Comment:
What does debugging do and should that be left on?

Reply:
Oh wait, ops, I will fix that

Reply:
Looks good otherwise